akka-docs/src/main/paradox/common/cluster-background.md

```diff
@@ -0,0 +1,72 @@
+# When and where to use Akka Cluster
```

Comment:
Good to have this as a separate page. We need a way to redirect or keep the original URL, which I think is shared widely out there. Can we redirect an anchor?
https://doc.akka.io/docs/akka/common/cluster-usage.html#when-and-where-to-use-akka-cluster

Reply:
You can do the anchor in the second part of the redirect but for a request including the fragment (anchor) in the first, this value is never sent to the server. We of course do not want to redirect all requests for the current `cluster-usage.html` to the new when/where page. Therefore, I think the best we can do, which is not ideal, is retain the old header for in the classic page and add a link under it to the new.

Reply:
On the original page:
```
## When and where to use Akka Cluster
 
See [Choosing Akka Cluster](typed/choosing-cluster.md#when-and-where-to-use-akka-cluster).
```

Reply:
sounds good to me